conda/recipe/meta.yaml

```
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 package:
-  name: nvidia-dali-cu{{ environ.get('CUDA_VERSION', '') }}
+  name: nvidia-dali-cu{{ environ.get('CUDA_VERSION_DIGIT_ONLY', '') }}
```

```suggestion
  name: nvidia-dali-cu{{ environ.get('CUDA_VERSION_STR', '') | replace(".","") }}
```
And get rid of CUDA_VERSION_DIGIT_ONLY at all (but I'm not 100% sure if above would work, but it should).